spec/cucumber/cli/rerun_spec.rb

```diff
@@ -71,16 +71,17 @@ module Cli
         end
       end
 
-      context "contains features and scenarios with and without spaces same line and across lines" do
+      context "contains features and scenarios with and without spaces on same line and across lines" do
```

Comment:
there are a lot of rules in this example! I think it would be easier to read if you split it into several examples.